docs/rackhd/ubuntu_source_installation.rst

```diff
@@ -9,61 +105,154 @@ We will leverage the ansible roles created for the RackHD demonstration environm
     sudo apt-get install git
     sudo apt-get update
     sudo apt-get dist-upgrade
-    sudo reboot
-
-    cd ~
-    git clone https://github.com/rackhd/rackhd
     sudo apt-get install ansible
-    cd ~/rackhd/packer/ansible
-    ansible-playbook -i "local," -K -c local rackhd_local.yml
+    sudo apt-get install -y g++ libkrb5-dev unzip
```

Comment:
the reason why you give up ansible-playbook?

Reply:
The ansible-playbook doesn't work well. It occur some errors, such as 
```
TASK [mongodb : Install MongoDB] ***********************************************
failed: [local] => (item=[u'mongodb=1:2.6.10-1ubuntu2']) => {"cache_update_time": 0, "cache_updated": false, "failed": true, "item": ["mongodb=1:2.6.10-1ubuntu2"], "msg": "'/usr/bin/apt-get -y -o \"Dpkg::Options::=--force-confdef\" -o \"Dpkg::Options::=--force-confold\"   install 'mongodb=1:2.6.10-1ubuntu2'' failed: E: Version '1:2.6.10-1ubuntu2' for 'mongodb' was not found\n", "stderr": "E: Version '1:2.6.10-1ubuntu2' for 'mongodb' was not found\n", "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\n", "stdout_lines": ["Reading package lists...", "Building dependency tree...", "Reading state information..."]}

PLAY RECAP *********************************************************************
local                      : ok=5    changed=0    unreachable=0    failed=1
```
@changev